mor/mor/feel/feelmor/parameterspace.hpp

```diff
@@ -219,7 +219,10 @@ class ParameterSpace: public CommObject, public std::enable_shared_from_this<Par
                 if (value >= min_val && value <= max_val)
                     this->operator()( it - paramNames.begin() ) = value;
                 else
+                {
+                    Feel::cout << tc::red << "Warning: " << tc::reset << value << " not in range [" << min_val << ", " << max_val << "]" << std::endl;
```

Comment:
Warning Feel::cout use the environment worldcomm, and can not show the message if a sub communicator is used. LOG( Warning ) will do the job. 
However, here we can stop the application? with an exception thrown

Reply:
I added the display with `Feel::cout` because sometimes I forgot to check the log and spent some time to find that the issue what that I was setting parameters outside the range !
But yes we may want to throw an exception. @prudhomm what do you think ?